homeassistant/components/alexa/smart_home.py

```diff
@@ -34,6 +36,17 @@
     TEMP_CELSIUS: 'CELSIUS',
 }
 
+API_THERMOSTAT_MODES = {
+    climate.STATE_HEAT: 'HEAT',
+    climate.STATE_COOL: 'COOL',
+    climate.STATE_AUTO: 'AUTO',
+    climate.STATE_ECO: 'ECO',
+    climate.STATE_IDLE: 'OFF',
+    climate.STATE_FAN_ONLY: 'OFF',
+    climate.STATE_DRY: 'OFF',
```

Comment:
I wonder with these kind of values that we can't map, should we make up something like this or can we return some "Unknown" or "null" value ?

Reply:
The Alexa.ThermostatController spec doesn't seem to include any placeholder values or errors for unknown state. For the latter two states, mapping to `OFF` makes sense to me (since heating/cooling is off and this is just an auxiliary operational mode of the device). The reverse direction isn't true, though, so I should probably just allow the first mapping from `OFF` to `climate.STATE_IDLE` only.